client/src/main/java/com/battleships/start_window/data_insertion/SettingsSeekBarsData.java

```diff
@@ -1,10 +1,11 @@
-package com.battleships.start_window.InsertData;
+package com.battleships.start_window.data_insertion;
+
 import com.battleships.Translator;
 import javafx.fxml.FXML;
 import javafx.scene.control.Label;
 import javafx.scene.control.Slider;
 
-public class SeekBarsDataInserter {
+public class SettingsSeekBarsData {
```

Comment:
Unclear class name

Reply:
We think that this name is a good name. Can you propose better? SettingsSeekBarsData affects all seeking bars in app menu.